tests/test_density_GaussianDensity.py

```diff
@@ -78,61 +78,64 @@ def test_change_box_dimension(self):
         with pytest.raises(ValueError):
             gd.compute((test_box, test_points))
 
-    def test_sum_2d(self):
+    num_points = ("num_points", [1, 10, 100])
+
+    @pytest.mark.parametrize(*num_points)
+    def test_sum_2d(self, num_points):
```

Comment:
It's a little strange to shadow the previously declared `num_points` with the function parameter `num_points`. I would try to choose distinct names here to avoid confusion.